### DISCUSSION TITLE
Paulmann SmartHome Zigbee LED module coin 1x2.5W RGBW (929.66)

### DISCUSSION THREAD

**Discussion by MightyBob** [Koenkk/zigbee2mqtt#5569] [Devices]
Hi,
I'm working on integration of the SmartHome Zigbee LED module coin 1x2.5W RGBW (ref. 929.66) in my zigbee network.
As this device is not currently supported, I've created the description file and got the jpeg image, based on a similar device (500.47).
My testing are still in progress, until now everything is working smoothly.
That's why I'm posting my work here for integration in the project branch.

[929.66.zip](https://github.com/Koenkk/zigbee2mqtt/files/5769827/929.66.zip)

This is the addon to devices.js :
`    {
        zigbeeModel: ['929.66', 'H036-0007'],
        model: '929.66',
        vendor: 'Paulmann',
        description: 'Smart Home Zigbee LED module coin 1x2.5W RGBW',
        extend: preset.light_onoff_brightness_colortemp_colorxy,
    },`

I hope this will help the community... and sorry if there are some errors/issues, I'm still a beginer ;-)

**Comment by Koenkk** [comment]
Can you share the data/database.db entry of this device?

**Comment by MightyBob** [comment]
Sure! Here we go:

`{"id":4,"type":"Router","ieeeAddr":"0x00158d0000000000","nwkAddr":63000,"manufId":4624,"manufName":"Paulmann Licht","powerSource":"DC Source","modelId":"H036-0007","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":269,"inClusterList":[0,4,3,6,8,5,768,4096],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"H036-0007","manufacturerName":"Paulmann Licht","powerSource":4,"zclVersion":3,"appVersion":14,"stackVersion":1,"hwVersion":1,"dateCode":"20190515","swBuildId":"1400-0001"}},"genOnOff":{"attributes":{"onOff":1}},"genLevelCtrl":{"attributes":{"currentLevel":254}},"lightingColorCtrl":{"attributes":{"colorTemperature":370,"currentX":30295,"currentY":27192}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[33],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":14,"stackVersion":1,"hwVersion":1,"dateCode":"20190515","swBuildId":"1400-0001","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1609887377545}`

Note: IEEE and network addresses has been anonymized ;-)

**Reply by Koenkk** [reply]
Added, thanks!

Changes will be available in the latest dev branch in a few hours (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)
